sites/kit.svelte.dev/src/routes/home/Video.svelte

```diff
@@ -46,6 +53,19 @@
 			observer.disconnect();
 		};
 	});
+
+	/**
+	 * @param {HTMLTrackElement} node
+	 */
+	function set_cue_position(node) {
+		const cues = node.track.cues;
+		if (!cues) return;
```

Comment:
I suspect this works because the .vtt file is loaded by the time the JS runs (basically the inverse of the problem in `onMount` here), but it _could_ presumably happen in the opposite order. Is there an event we can listen for if `cues` doesn't exist yet?

Reply:
Good call - I was able to listen to the `<track>`'s `load` event if we don't have cues yet. I tested it by wrapping the `<track>` in an `{#if browser}` block and it seemed to work. https://github.com/sveltejs/kit/pull/8069/commits/922677fdf7712786f20ee8be8ad5f18d67f30736